.gitmodules

```diff
@@ -1,12 +1,6 @@
-[submodule "firmware/src/deps/attiny-common"]
-	path = firmware/src/deps/attiny-common
-	url = git@github.com:tessel/attiny-common.git
```

Comment:
I'm not sure why this submodule is listed. `attiny-common` is a Node library listed in the `dependencies` of this `package.json`. I see no reason why it should be a submodule.